user-service/src/main/java/hcmus/alumni/userservice/controller/UserServiceController.java

```diff
@@ -353,4 +373,110 @@ public ResponseEntity<String> updateAlumniVerificationStatus(@PathVariable Strin
 		
 		return ResponseEntity.ok("Alumni verification approved successfully");
 	}
+	
+
+	@PreAuthorize("hasAnyAuthority('User.Create')")
+	@PostMapping("/create-user")
+    public ResponseEntity<String> adminCreateUser(
+            @RequestParam String email,
+            @RequestParam String role,
+            @RequestParam String fullName) {
```

Comment:
```suggestion
    public ResponseEntity<String> adminCreateUser(
            @RequestBody UserModel req) {
```
dùng `@RequestBody` r sau đó lọc lại chỉ lấy email, role, và fullName